src/components/header/header.js

```diff
@@ -20,6 +20,6 @@ export const Header = ({ name, Logo, MenuItems, ProfileItems }) => {
 Header.propTypes = {
   name: PropTypes.string,
   Logo: PropTypes.element.isRequired,
-  MenuItems: PropTypes.array.isRequired,
-  ProfileItems: PropTypes.array
+  MenuItems: PropTypes.element.isRequired,
+  ProfileItems: PropTypes.element
```

Comment:
Nice catch. I think `node` is a better type. We could pass a plain `<a>` tag which is not necessarily a react element
```suggestion
  MenuItems: PropTypes.node.isRequired,
  ProfileItems: PropTypes.node
```